addons/mail/models/discuss/discuss_gif_favorite.py

```diff
@@ -17,3 +17,9 @@ class GifFavorite(models.Model):
             "User should not have duplicated favorite GIF",
         ),
     ]
+
+    @api.returns('self', lambda value: value.id)
+    def copy(self, default=None):
+        default = dict(default or {},
+                       tenor_gif_id=f"{self.tenor_gif_id} (copy)")
```

Comment:
Does not make sense, the id has meaning. Adding copy make it an invalid id.
If anything there should be an sql unique constraint on (create_uid, tenor_gif_id)

Reply:
Indeed, copying makes no sense ^^ .

Reply:
thanks for feedback @tde @thir,
how about adding copy="0" in the form, as constrain is raised on clicking copy from action

Reply:
adding copy="0" in form is cleaner

Reply:
Yes, it if is possible to remove the copy button, please do so. Thanks!